reference/speaker.md

```diff
@@ -60,106 +60,59 @@ It is possible to stop all the sounds currently playing in a speaker by setting
 
 **Name**
 
-**wb\_speaker\_set\_language**, **wb\_speaker\_get\_language**, **wb\_speaker\_speak** - *perform text-to-speech*
+**wb\_speaker\_set\_engine**, **wb\_speaker\_set\_language**, **wb\_speaker\_get\_engine**, **wb\_speaker\_get\_language**, **wb\_speaker\_speak** - *perform text-to-speech*
 
 {[C++](cpp-api.md#cpp_speaker)}, {[Java](java-api.md#java_speaker)}, {[Python](python-api.md#python_speaker)}, {[Matlab](matlab-api.md#matlab_speaker)}, {[ROS](ros-api.md)}
 
 ```c
 #include <webots/speaker.h>
 
+bool wb_speaker_set_engine(WbDeviceTag tag, const char *engine);
 void wb_speaker_set_language(WbDeviceTag tag, const char *language);
```

Comment:
Since the available languages can vary from one computer to another one, should we return a boolean to indicate success or not?